### DISCUSSION TITLE
MultiModalBeamforming Example audio files are missing

### DISCUSSION THREAD

**Discussion by batwood001** [speechbrain/speechbrain#1832] [General]
In the [MultiModal Beamforming Example](https://colab.research.google.com/drive/1UVoYDUiIrwMpBTghQPbA6rC1mc9IBzi6?usp=sharing#scrollTo=leHtExv6LkUg), the audio files (e.g. https://www.dropbox.com/s/1bag9h3q588bjxx/speech_-0.82918_0.55279_-0.082918.flac) are missing / deleted from Dropbox. Anyone have a copy? 🙏

**Comment by FrancoisGrondin** [comment]
Hello, I have made a change in the Jupyter notebook and updated the path. Thanks for the heads up!
